crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -25,6 +25,9 @@ enum PipelineDescriptor {
     ComputePipelineDescriptor(Box<ComputePipelineDescriptor>),
 }
 
+/// A pipeline used to send data to the GPU.
```

Comment:
This is a mischaracterization. Pipelines aren't really about sending data to the GPU, they are about defining the data layout and shader logic for a specific GPU task. See https://docs.rs/wgpu/latest/wgpu/struct.RenderPipeline.html for details.

Reply:
Ah, thanks! This is my fault, and I was hoping someone with more rendering expertise would double check.

Reply:
Fixed. Sorry I had forgotten about this PR.